diesel_codegen/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(warnings)]
+#![recursion_limit = "512"]
```

Comment:
What is this needed for? (It's fine, just maybe add a comment what requires this, so we can remove it when it's no longer needed.)